### PR TITLE
feat: Add phase descriptions and correct loaders

### PR DESCRIPTION
**Pull request by hofmeister**
image
image

### CODE REVIEW

File: stories/data/initializeMocks.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ BlockTypeProvider.register({
 
 BlockTypeProvider.register({
     kind: 'kapeta/block-type-gateway-http',
-    version: '2.2.0',
+    version: '1.3.6',
```
Comment (andertun) [inline stories/data/initializeMocks.ts:49]: Was this downgrade intentional?

**Review by andertun — APPROVED** [review]
It I understand the screenshot correctly it looks like the DonutProgressIcon becomes gray when that thing is done? I think we should set opacity to 0 for the DonutProgressIcon it when it is done so we don't have gray circles there

File: stories/data/initializeMocks.ts (hunk 1 above)
Comment (hofmeister) [reply stories/data/initializeMocks.ts:49]: Yea it's just mock data - and I had to generate new events . These versions have no consequence - just needs to match the events data

**Comment by hofmeister** [comment]
> If I understand the screenshot correctly it looks like the DonutProgressIcon becomes gray when that thing is done? I think we should set opacity to 0 for the DonutProgressIcon (or remove it) when it is done so we don't have gray circles there

Not sure where you're seeing those ? I think the screenshot just caught them at the right time.

**Comment by andertun** [comment]
> > If I understand the screenshot correctly it looks like the DonutProgressIcon becomes gray when that thing is done? I think we should set opacity to 0 for the DonutProgressIcon (or remove it) when it is done so we don't have gray circles there
> 
> Not sure where you're seeing those ? I think the screenshot just caught them at the right time.

Ah, I see I already took care of the opacity thing inside `AIMessageAccordion`

**Pull request merged**